.github/workflows/ci.yml

```diff
@@ -94,6 +94,10 @@ jobs:
       run: ./x new template
 
     - name: build project
-      if: ${{ matrix.target.platform != 'linux' || matrix.host == 'ubuntu-latest' }}
+      if: |
+        ${{ matrix.host == 'macos-latest' && matrix.target.platform != 'linux' }} ||
+        ${{ matrix.host == 'windows-latest' && matrix.target.platform != 'linux' }} ||
+        ${{ matrix.host == 'windows-latest' && matrix.target.platform != 'macos' }} ||
+        ${{ matrix.host == 'windows-latest' && matrix.target.platform != 'ios' }}
```

Comment:
So macos can target anything _but_ linux, and windows can only target windows and android?

Reply:
Re. recent push: would it be cleaner to combine all the statements for `host=windows-latest`? Maybe with [`contains()`](https://docs.github.com/en/actions/learn-github-actions/expressions#contains)?

Reply:
yes. windows can't target macos/ios because it doesn't support posix symlinks. someone could probably work around this if so inclined. only linux can target linux because it's not clear what linux is exactly. glibc/musl, gtk/qt, wayland/x11.

Reply:
True, I don't care about supporting macos/ios/linux cross-compile from Windows (nor anything for that matter). Feel free to yeet this as a comment into CI for future readers though :)

Reply:
https://docs.github.com/en/actions/learn-github-actions/expressions#example-matching-an-array-of-strings ugh `fromJson` makes this pretty retarded.